### PR TITLE
add array contains

### PR DESCRIPTION
**Pull request by eleith**
add https://firebase.google.com/docs/firestore/query-data/queries#array_membership (array contains) query operator

### CODE REVIEW

File: definitions/npm/firebase_v5.x.x/flow_v0.34.x-/test_firebase.js [hunk 1]
```diff
@@ -1,305 +1,322 @@
...
+  .collection("/foo")
+  .where("id", "array-contains", "5")
+  .get({ source: "cache" });
+
+firebase
+  .firestore()
+  .collection("/foo")
+  // $ExpectError
+  .where("id", "lte", "5")
+  .get({ source: "cache" });
```
Comment (AndrewSouthpaw) [inline definitions/npm/firebase_v5.x.x/flow_v0.34.x-/test_firebase.js]: Special bonus points to the first person that refactors this test file using `describe/it` blocks. 😉  (Not required for this PR.)

File: definitions/npm/firebase_v5.x.x/flow_v0.34.x-/test_firebase.js [hunk 2]
```diff
@@ -1,305 +1,322 @@
 // @flow
 
-import firebase from 'firebase'
-import 'firebase/firestore'
-import app from '@firebase/app'
-import auth from 'firebase/auth'
-import database from 'firebase/database'
+import firebase from "firebase";
```
Comment (AndrewSouthpaw) [inline definitions/npm/firebase_v5.x.x/flow_v0.34.x-/test_firebase.js]: All of these look like changes that I *think* aren't consistent with our current prettier setup.. can you verify?

Comment (eleith) [reply definitions/npm/firebase_v5.x.x/flow_v0.34.x-/test_firebase.js]: my bad. for some reason my editor is linting/prettifying based on my personal preferences and not using the preferences of this repo. will revert.

File: definitions/npm/firebase_v5.x.x/flow_v0.34.x-/test_firebase.js (hunk 1 above)
Comment (eleith) [reply definitions/npm/firebase_v5.x.x/flow_v0.34.x-/test_firebase.js]: i'm tempted. the firebase library still needs some updating and it hasn't been straight forward to add tests. what is a well laid out test that you would use as a model?

also any advice on how to get the eslint/prettier rules to pick up? my editor keeps turing the single quotes to double quotes and i'm confused...

**Comment by GAntoine** [comment]
Merging in now that Andrew's comments were addressed.

**Pull request merged**